streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamKTableJoinTest.java

```diff
@@ -135,4 +137,104 @@ public void testJoin() {
 
         processor.checkAndClearProcessResult("2:XX2+YY2", "3:XX3+YY3");
     }
+
+
+    @Test
+    public void testGlobalKTableJoin() throws Exception {
```

Comment:
Not part of this JIAR, but should be rewrite this (and the existing) test to a proper unit test (-> `KStreamKTableJoinProcessorTest`)? If you don't want to rewrite the test it's ok, and we create a JIRA for it.

Reply:
Looking at the `org.apache.kafka.streams.kstream.internals` test package it looks like all
tests are named after the operation type rather than the processor name. Switching to
proper unit test style would be a big change and would be better to do across the board rather than just for the one processor touched by this PR.